source/Interface.h

```diff
@@ -182,6 +176,41 @@ class Interface {
 		Truncate truncate = Truncate::NONE;
 	};
 
+	// This class handles "label", "string", "button" and "dynamic button" elements.
+	class BasicTextElement : public TextElement {
+	public:
+		BasicTextElement(const DataNode &node, const Point &globalAnchor);
+
+	protected:
+		// Parse the given data line: one that is not recognized by Element
+		// itself. This returns false if it does not recognize the line, either.
+		virtual bool ParseLine(const DataNode &node) override;
+		// Report the actual dimensions of the object that will be drawn.
+		virtual Point NativeDimensions(const Information &info, int state) const override;
+		// Draw this element in the given rectangle.
+		virtual void Draw(const Rectangle &rect, const Information &info, int state) const override;
+	};
+
+	// This class handles "wrapped label", "wrapped string",
+	// "wrapped button", and "wrapped dynamic button" elements.
+	class WrappedTextElement : public TextElement {
+	public:
+		WrappedTextElement(const DataNode &node, const Point &globalAnchor);
+
+	protected:
+		// Parse the given data line: one that is not recognized by Element
+		// itself. This returns false if it does not recognize the line, either.
+		virtual bool ParseLine(const DataNode &node) override;
+		// Report the actual dimensions of the object that will be drawn.
+		virtual Point NativeDimensions(const Information &info, int state) const override;
+		// Draw this element in the given rectangle.
+		virtual void Draw(const Rectangle &rect, const Information &info, int state) const override;
+
+	private:
+		mutable WrappedText text;
+		Alignment alignment = Alignment::LEFT;
```

Comment:
The class `Interface::Element` has a field `Point alignment`. It's not the same type as this, so the compiler can probably handle it, but humans will probably do better with a different name. `textAlignment`, perhaps.